app/src/main/java/org/oppia/android/app/profile/AdminAuthActivityPresenter.kt

```diff
@@ -58,8 +57,7 @@ class AdminAuthActivityPresenter @Inject constructor(
     }
 
     binding.adminAuthInputPinEditText.setOnEditorActionListener { _, actionId, event ->
-      if (actionId == EditorInfo.IME_ACTION_DONE ||
-        (event != null && (event.keyCode == KeyEvent.KEYCODE_ENTER))
+      if (event != null && (event.keyCode == KeyEvent.KEYCODE_ENTER)
```

Comment:
Are we able to use the keyboard action done,  if we input the correct pin in the input field? or Do we have to only use the submit button?

Reply:
Yes, we can use the keyboard button. It same as the submit button now.
Until and unless the length of the pin entered is less than the adminPin's length the keyboard submit can be clicked but would have no action.

Reply:
> but would have no action.
Will this show any error message or  nothing will happen?

Reply:
No, it won't show. It will act as ditto as the submit button, so nothing will happen.